eosio.msig/abi/eosio.msig.abi

```diff
@@ -108,6 +114,13 @@
         {"name": "requested_approvals", "type": "permission_level[]"},
         {"name": "provided_approvals", "type": "permission_level[]"}
       ]
+    },{
+      "name": "invalidation",
+      "base": "",
+      "fields": [
+        {"name": "account",                "type": "account_name"},
+        {"name": "last_invalidation_time", "type": "uint64"}
```

Comment:
Please use `time_point` for times rather than a raw `uint64`. This can be set to the current time after converting the return value of the `current_time` intrinsic to a `time_point` type (see `system_contract::current_time_point()` for how to do this).